quast/5.2.0/Dockerfile

```diff
@@ -0,0 +1,110 @@
+ARG QUAST_VER="5.2.0"
+
+## Builder ##
+FROM ubuntu:focal as builder
+
+ARG QUAST_VER
+
+# define timezone to avoid build stalls
+ENV TZ=America/New_York
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+RUN apt-get update && apt-get install --no-install-recommends -y \
+    wget \
+    python2 \
+    perl \
+    cpanminus \
+    g++ \
+    make \
+    openjdk-8-jre-headless \
+    r-base \
+    pkg-config \
+    libfreetype6-dev \
+    libpng-dev \
+    libboost-all-dev \
+    locales &&\
+    locale-gen en_US.UTF-8 &&\
+    cpanm Time::HiRes &&\
+    apt-get autoclean &&\
+    rm -rf /var/lib/apt/lists/*
+
+# python dependencies
+RUN update-alternatives --install /usr/bin/python python /usr/bin/python2 1 &&\
+    wget https://bootstrap.pypa.io/pip/2.7/get-pip.py && python get-pip.py &&\
+    pip install --no-cache matplotlib simplejson joblib
+
+# install quast
+RUN wget https://github.com/ablab/quast/releases/download/quast_${QUAST_VER}/quast-${QUAST_VER}.tar.gz && \
+    tar -xzf quast-${QUAST_VER}.tar.gz && \
+    rm -rf quast-${QUAST_VER}.tar.gz && \
+    cd /quast-${QUAST_VER} && \
+    /quast-${QUAST_VER}/setup.py install
+
+# add GRIDSS for SV detection
+ADD https://github.com/ablab/quast/raw/master/external_tools/gridss/gridss-1.4.1.jar /quast-${QUAST_VER}/quast_libs/gridss/
```

Comment:
Do you think it would be helpful to list the gridss version near the top in an ARG instruction?